oe_theme.theme

```diff
@@ -1423,6 +1423,10 @@ function oe_theme_preprocess_paragraph__oe_banner(array &$variables): void {
   $variables['url'] = Url::fromUri($link['uri']);
   $variables['label'] = $link['title'];
 
+  if ($paragraph->get('field_oe_banner_full_width')->value === 'yes') {
+    $variables['full_width'] = TRUE;
+  }
```

Comment:
I'm not sure about this. I know that in the previous oe_paragraphs ticket it was asked to do list_text with Yes/No, but actually looking at the solution here it's meant to be bool. Using strings for deciding a bool is not correct. I checked with @ademarco and we agreed to have a checkbox instead on the paragraph. Please apply the change on the paragraphs side within this ticket so we make this right. :)

Reply:
Did the change but boolean field returns 1/0 so it's the same approach.